glossarpy/GlossTxt.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+
+class GlossTxt:
+    '''Handles RST-specific output'''
+
+    def underline_text(self, text:str, underlinechar="-"):
```

Comment:
Typing should be consistent throughout.  I.E.  `def underline_text(self, text: str, underlinechar: str = "-"):`